spec/factories/user_phenotypes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user_phenotype do
+    association :user
+    association :phenotype
+    variation "pink"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.